p2p/module_raintree_test.go

```diff
@@ -367,85 +363,75 @@ func prepareConnMock(t *testing.T, expectedNumNetworkReads, expectedNumNetworkWr
 	return connMock
 }
 
-func prepareP2PModules(t *testing.T, configs []modules.Config) (p2pModules map[string]*p2pModule) {
-	p2pModules = make(map[string]*p2pModule, len(configs))
-	for i, config := range configs {
-		createTestingGenesisAndConfigFiles(t, config, modules.GenesisState{}, i)
-		p2pMod, err := Create(testingConfigFilePath+strconv.Itoa(i)+jsonPosfix, testingGenesisFilePath+jsonPosfix, false)
+func prepareP2PModules(t *testing.T, runtimeConfigs []modules.RuntimeMgr) (p2pModules map[string]*p2pModule) {
+	p2pModules = make(map[string]*p2pModule, len(runtimeConfigs))
+	for i, runtimeConfig := range runtimeConfigs {
+		p2pMod, err := Create(runtimeConfig)
 		require.NoError(t, err)
 		p2pModules[validatorId(t, i+1)] = p2pMod.(*p2pModule)
 	}
 	return
 }
 
-func createTestingGenesisAndConfigFiles(t *testing.T, cfg modules.Config, genesisState modules.GenesisState, n int) {
-	config, err := json.Marshal(cfg.P2P)
-	require.NoError(t, err)
-
-	genesis, err := json.Marshal(genesisState.ConsensusGenesisState)
-	require.NoError(t, err)
-
-	genesisFile := make(map[string]json.RawMessage)
-	configFile := make(map[string]json.RawMessage)
-	moduleName := new(p2pModule).GetModuleName()
-
-	genesisFile[test_artifacts.GetGenesisFileName(moduleName)] = genesis
-	configFile[moduleName] = config
-	genesisFileBz, err := json.MarshalIndent(genesisFile, "", "    ")
-	require.NoError(t, err)
-
-	p2pFileBz, err := json.MarshalIndent(configFile, "", "    ")
-	require.NoError(t, err)
-	require.NoError(t, ioutil.WriteFile(testingGenesisFilePath+jsonPosfix, genesisFileBz, 0777))
-	require.NoError(t, ioutil.WriteFile(testingConfigFilePath+strconv.Itoa(n)+jsonPosfix, p2pFileBz, 0777))
-}
-
-func createConfigs(t *testing.T, numValidators int) (configs []modules.Config, genesisState modules.GenesisState) {
-	configs = make([]modules.Config, numValidators)
+func createMockRuntimeMgrs(t *testing.T, numValidators int) []modules.RuntimeMgr {
```

Comment:
Blocker:

Please add comment around any `Mock` creation, see the other mocks in this file for example around expected behaviors